packages/common/src/configuration/service-configuration.ts

```diff
@@ -141,6 +143,17 @@ export class ServiceConfiguration {
                     default: 5,
                     doc: 'Timeout value after which the task has to be terminated',
                 },
+                retentionTimeInDays: {
+                    format: 'int',
+                    default: 3,
+                    doc:
+                        'The minimum time to retain the working directory for the task on the compute node where it ran, from the time it completes execution. After this time, the Batch service may delete the working directory and all its contents.',
+                },
+                maxTaskRetryCount: {
```

Comment:
what is the default value when we don't specify any value?

Reply:
No retry if nothing is specified and it keep working directory for 7 days if nothing is soecified.
https://docs.microsoft.com/en-us/azure/batch/batch-task-fail-event
https://docs.microsoft.com/en-us/rest/api/batchservice/task/add#taskconstraints

Reply:
Yes default is no retry and 7 days to keep working files